modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc

```diff
@@ -8,6 +8,10 @@
 
 {gitops-title} Operator automatically creates a ready-to-use Argo CD instance that is available in the `openshift-gitops` namespace.
 
+{gitops-title} automatically creates a ready-to-use Argo CD instance that is available in the `openshift-gitops` namespace. Optionally, you can install and deploy a new Argo CD instance to manage cluster configurations or deploy applications.
```

Comment:
You seem to be saying the same sentence twice here?

Reply:
Thanks for pointing this out. Fixed.